packages/plugin-pack/sources/commands/pack.ts

```diff
@@ -39,7 +42,11 @@ export default class PackCommand extends BaseCommand {
     if (!workspace)
       throw new WorkspaceRequiredError(this.context.cwd);
 
-    const target = ppath.resolve(workspace.cwd, toFilename(`package.tgz`));
+    const archiveName = this.nameArchive
+      ? `${prettyWorkspaceSlug(workspace)}.tgz`
```

Comment:
Tried using `structUtils.slugifyLocator` but that creates `@bery-core-exotic` without any version attached. Not sure if this is something that I have to change in the test or if this is expected. It would be sufficient for my specific use case but if you would want to have multiple versions of an archive in a folder you would loose readability. 

In any case I'm open to alternative formats.